### PR TITLE
Only static code analysis in Travis Build Matrix

### PR DESCRIPTION
**Pull request by lekshmideepu**
Except the status code analysis check, removed all the other matrix builds from Travis CI. Currently the static checks for changed files are not implemented in our githubactions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -201,37 +147,4 @@ before_script:
...
-#  region: eu-central-1
-#  skip_cleanup: true
-#  on:
-#    repo: nest/nest-simulator
-#    branch: master
-#    condition: $xNEST_BUILD_TYPE != STATIC_CODE_ANALYSIS
-#  local-dir: "$TRAVIS_BUILD_DIR/build/artefacts_upload"
-#  upload-dir: "$TRAVIS_REPO_SLUG/$TRAVIS_BUILD_NUMBER/$TRAVIS_JOB_NUMBER"
-#  acl: bucket_owner_full_control
+   - ./extras/travis_build.sh 2>&1 | tee travis_build.sh.log
```
Comment (heplesser) [inline .travis.yml:150]: Add a newline.
```suggestion
   - ./extras/travis_build.sh 2>&1 | tee travis_build.sh.log
   
```

**Review by heplesser — CHANGES_REQUESTED** [review]
@lekshmideepu Thank you! As long as Github does not actually perform the static analysis, would you remove that from the Github actions matrix to avoid confusing false positives? And one tiny suggestion, see below.

**Comment by lekshmideepu** [comment]
@heplesser : Thanks for taking a look. I have just addressed both your comments.

**Review by heplesser — APPROVED** [review]
Thanks!

**Review by jougs — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**